overlord/hookstate/hooktest/handler.go

```diff
@@ -0,0 +1,78 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package hooktest
+
+import "fmt"
+
+// MockHandler is a mock hookstate.Handler.
+type MockHandler struct {
+	BeforeCalled bool
+	BeforeError  bool
+
+	DoneCalled bool
+	DoneError  bool
+
+	ErrorCalled bool
+	ErrorError  bool
+	Err         error
+}
+
+// NewMockHandler returns a new MockHandler.
+func NewMockHandler() *MockHandler {
+	return &MockHandler{
```

Comment:
Pardon my ignorance here, but given goes default policy of zeroing values, why explicatively init them here? Or am I missing something?